.changeset/plenty-starfishes-invent.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": patch
```

Comment:
Should this PR be considered as minor instead of patch because it makes a change of the part of the public API?